config.py

```diff
@@ -0,0 +1,14 @@
+import os
+
+SECRET_KEY = 'vhnvgcgxfzgyjcfdhtdcxz5w6e43rthr53x5twt'
+
```

Comment:
It is a good practice to either put secret keys in the environment or in file ignored by git.